browser_use/agent/message_manager/service.py

```diff
@@ -225,7 +225,19 @@ def _init_messages(self) -> None:
 		self._add_message_with_tokens(task_message, message_type='init')
 
 		if self.settings.sensitive_data:
-			info = f'Here are placeholders for sensitive data: {list(self.settings.sensitive_data.keys())}'
+			sensitive_data = self.settings.sensitive_data
```

Comment:
ignore this code as it passes all the sensitive data regardless of passing and checking current url with domain url of sensitive data.